docs/reference/completions.md

```diff
@@ -93,6 +93,12 @@ is identical to the `contents`:
   slightly grayed
   and does not affect the trigger itself.
 
+  Annotations can also be defined using `annotation`:
```

Comment:
This should be in a separate `annotation` definition that mentions the version this was added in.